src/many-modules/src/_4_events.rs

```diff
@@ -834,4 +834,18 @@ mod tests {
         assert_eq!(list_returns.nb_events, 1);
         assert_eq!(list_returns.events.len(), 1);
     }
+
+    #[test]
+    fn encode_decode() {
+        let event = hex::decode("a30045030000000201c11a62e7dcea02a500820982010301d92710582080e8acda9634f6f745be872b0e5e9b65b1d3624a3ba91c6432143f60e90000020245020000000103f604d92712a301d92710582080e8acda9634f6f745be872b0e5e9b65b1d3624a3ba91c6432143f60e900000204a3003a00015f9001781d4163636f756e742077697468204944207b69647d20756e6b6e6f776e2e02a162696478326d6166666261686b736477617165656e6179793267786b65333268676237617134616f3477743734356c7366733677696a7005c11a62e7dcf5").unwrap();
```

Comment:
Please make multiple lines with this string

Reply:
addressed